samples/bluetooth/unicast_audio_server/boards/nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -1,2 +1,4 @@
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
```

Comment:
@Thalley For a specific configuration, and the defconfig of sysworkq is not set, only this one is found.

Reply:
Hmm, since this change will only affect a subset of configuration (e.g. whenever nRF5340 is used), perhaps it's not worth going through all samples and add this (even though we should to retain the existing functionality). Perhaps it is just better to use the warning so that the users of those samples can either set the stack size manually, or add a board configuration.